template/src/lib/message-validator.js

```diff
@@ -0,0 +1,6 @@
+const AsyncApiValidator = require('asyncapi-validator');
+
+module.exports.validateMessage = async (payload, topic,name,operation)=> {
+  let va = await AsyncApiValidator.fromSource('./asyncapi.yaml', {msgIdentifier: 'name'});
+  va.validate(name, payload, topic, operation);
```

Comment:
please catch an error and format a log message nicely, like some custom text "message rejected" + the reason from error message without the rest of stack trace